dashboard/src/app/teams/team-details/team-details.controller.ts

```diff
@@ -112,12 +112,12 @@ export class TeamDetailsController {
    */
   constructor(codenvyTeam: CodenvyTeam, codenvyResourcesDistribution: CodenvyResourcesDistribution, codenvyPermissions: CodenvyPermissions,
               codenvyUser: CodenvyUser, $route: ng.route.IRouteService, $location: ng.ILocationService, $rootScope: che.IRootScopeService,
-              $scope: ng.IScope, confirmDialogService: any, codenvyTeamEventsManager: CodenvyTeamEventsManager, cheNotification: any,
+              $scope: ng.IScope, confirmDialogService: any, codenvyOrganizationEventsManager: CodenvyTeamEventsManager, cheNotification: any,
```

Comment:
will not work,need to change : codenvyOrganizationEventsManager -> codenvyTeamEventsManager